Dockerfile.template

```diff
@@ -6,12 +6,12 @@ ENV MARIADB_VERSION=%%VERSION_FULL%%
 
 USER root
 
-RUN	apt-get update && \
```

Comment:
This gets removed because it's redundant. In modern Ubuntu, running `add-apt-repository` will automatically update the package index.

Reply:
Good call!